InnerEye/ML/dataset/scalar_dataset.py

```diff
@@ -747,13 +784,21 @@ def get_labels_for_imbalanced_sampler(self) -> List[float]:
         """
         return [item.label.item() for item in self.items]
 
-    def get_class_counts(self) -> Dict:
+    def get_class_counts(self) -> Dict[float, int]:
         """
-        Return class weights that are proportional to the inverse frequency of label counts.
+        Return the label counts.
         :return: Dictionary of {"label": count}
         """
-        all_labels = [item.label.item() for item in self.items]  # [N, 1]
-        return dict(Counter(all_labels))
+        if len(self.args.class_names) == 1:
+            return dict(Counter([item.label.item() for item in self.items if item.label.item() != 0]))
```

Comment:
I don't get that part. If you are feeding only labels of class 1 into `Counter`, you would only count positive labels, right? 
Also, is there a test for that function?

Reply:
Would also be helpful to have more detailed docu for the multilabel case, in particularly clarifying if the dictionary has class indices or names as keys.

Reply:
The intended behavior for binary classification tasks is to return the positive class count. I've added more details in the docstring. There are tests for this function in `test_scalar_dataset.py`.